src/test/java/kitchenpos/application/MenuServiceTest.java

```diff
@@ -0,0 +1,278 @@
+package kitchenpos.application;
+
+import kitchenpos.domain.*;
+import kitchenpos.infra.PurgomalumClient;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.math.BigDecimal;
+import java.util.*;
+import java.util.stream.Stream;
+
+import static java.util.Objects.*;
+import static kitchenpos.KitchenposFixture.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class MenuServiceTest {
+
+  private static final long POSITIVE_NUM = 1L;
+  private static final Long NEGATIVE_NUM = -1L;
+
+  @Mock
+  MenuRepository menuRepository;
+  @Mock
+  MenuGroupRepository menuGroupRepository;
+  @Mock
+  ProductRepository productRepository;
+  @Mock
+  PurgomalumClient purgomalumClient;
+
+  @InjectMocks
+  private MenuService menuService;
+
+  private static Stream<BigDecimal> menuPriceNullAndMinus() {
+    return Stream.of(
+            BigDecimal.valueOf(NEGATIVE_NUM),
+            null
+    );
+  }
+
+  @Test
+  @Order(1)
+  @DisplayName("가게 점주는 메뉴를 추가 할 수 있습니다.")
+  void addMenu() {
+    //given
+    Menu request = menu();
+    MenuGroup menuGroup = menuGroup();
+    Product product = product();
+    List<Product> productList = Collections.singletonList(product);
+
+    when(menuGroupRepository.findById(any())).thenReturn(Optional.of(menuGroup));
+    when(productRepository.findAllByIdIn(any())).thenReturn(productList);
+    when(productRepository.findById(any())).thenReturn(Optional.of(product));
+
+    //then
+    Assertions.assertDoesNotThrow(() -> {
+      Menu menu = menuService.create(request);
+    });
+  }
+
+  @Order(2)
+  @ParameterizedTest
+  @MethodSource("menuPriceNullAndMinus")
+  @DisplayName("메뉴의 가격이 존재하지 않거나 음수이면 IllegalArgumentException 예외 발생")
+  void price(BigDecimal price) {
+    //given
+    Menu request = menu();
+
+    //when
+    request.setPrice(price);
```

Comment:
메뉴 가격이 음수와 같이 자주 사용하거나 의미를 가지는 테스트 객체도 Fixture를 정의해보세요. 
저는 보통 아래와 같이 정의하곤 합니다 😄 

```suggestion
    Menu request = 음수_메뉴();
```